pkg/aws/db_documentdb.go

```diff
@@ -142,7 +149,8 @@ func getExpiredClusterSnapshots(svc rds.RDS) []*rds.DBClusterSnapshot {
 
 	if len(dbs) == 0 {
 		for _, snap := range snaps {
-			if snap.SnapshotCreateTime.Before(time.Now().UTC().Add(3 * time.Hour)) && common.CheckClusterSnapshot(snap) {
+			// do we need to force delete every snapshot on detroy command ?
```

Comment:
Don't know if we should trigger a force delete when `destroy` command for that kind of initial conditions based on timestamp ?

Reply:
=> ignore "add time" conditions for all those above structures